Makefile

```diff
@@ -7,7 +7,6 @@ PYTEST_COV_ARGS=--cov=$(PROJECT) --cov-config=../pyproject.toml \
 BLACK_FILES=$(PROJECT) doc/conf.py examples
 BLACKDOC_OPTIONS=--line-length 79
 DOCFORMATTER_FILES=$(PROJECT) doc/conf.py examples
-DOCFORMATTER_OPTIONS=--recursive --pre-summary-newline --make-summary-multi-line --wrap-summaries 79 --wrap-descriptions 79
 FLAKEHEAVEN_FILES=$(PROJECT) doc/conf.py examples
 LINT_FILES=$(PROJECT) doc/conf.py
```

Comment:
Should we keep 3 separate variables `BLACK_FILES`, `DOCFORMATTER_FILES` and `FLAKEHEAVEN_FILES` which hold the same thing?

Reply:
I think it's OK to combine them into a single variable `FORMAT_FILES`.

Reply:
Done in eb242ede97f317482b9634e37ebac44e720d6063